scrapyd_k8s.sample-docker.conf

```diff
@@ -5,6 +5,10 @@
 bind_address = 0.0.0.0
 http_port    = 6800
 
+## uncomment this section to enable authorization
+# username = questionmark
+# password = changeme123
```

Comment:
* no space between `#` and key, in-line with how this happens in the other sample config
* also add this to the Kubernetes sample config
* please use a more generic username here